core/src/main/java/org/apache/carbondata/core/segmentmeta/SegmentMetaDataInfoStats.java

```diff
@@ -86,28 +85,14 @@ public synchronized SegmentMetaDataInfo getTableSegmentMetaDataInfo(String table
   public synchronized void setBlockMetaDataInfo(String tableName, String segmentId,
       BlockColumnMetaDataInfo currentBlockColumnMetaInfo) {
     // check if tableName is present in tableSegmentMetaDataInfoMap
-    if (!this.tableSegmentMetaDataInfoMap.isEmpty() && null != this.tableSegmentMetaDataInfoMap
-        .get(tableName) && null != this.tableSegmentMetaDataInfoMap.get(tableName).get(segmentId)) {
-      // get previous blockColumn metadata information
-      BlockColumnMetaDataInfo previousBlockColumnMetaInfo =
-          this.tableSegmentMetaDataInfoMap.get(tableName).get(segmentId);
-      // compare and get updated min and max values
-      byte[][] updatedMin = BlockIndex.compareAndUpdateMinMax(previousBlockColumnMetaInfo.getMin(),
```

Comment:
why this code was removed ? This is for segment min max I guess

Reply:
This code came from Segment_Level MINMAX.

The segment_level minmax will be updated after new carbondata file be wrintten into the segment folder.
Now, it can be remove because update will write into new segment folder.

Reply:
ok